config/routes.rb

```diff
@@ -1,12 +1,15 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /en|vi/ do
     resources :microposts, only: %i(index)
+    resources :users, only: %i(new create show)
```

Comment:
```suggestion
    resources :users, only: %i(new show)
```
Thừa, action create đã khai báo trong /signup phía dưới r